### DISCUSSION TITLE
Second collection in same folder shows empty entries.

### DISCUSSION THREAD

**Discussion by onweru** [sveltia/sveltia-cms#171] [Q&A] [answered]
In hugo, it's common to have 
```
── _index.md
├── about-us.md
├── become-a-partner.md
```

The index page would be the list page with different frontmatter. On the cms, _index.md would not be shown in the same collection as the other pages in the same folder.

Here is what I have observed.

1. If the _index.md is added as file collection, it will show on the cms but all its fields will be empty. Completely ignoring the content in the file.
2. If it's added as a separate collection with a filter field, it will affect the other collection. That is if it's listed first in the cms config. In such a case, the second collection will show `this collections has no entries yet`.

Am I missing something?

**Comment by kyoshino** [comment]
I know Hugo’s `_index.md` can be a bit tricky. I have seen https://github.com/decaporg/decap-cms/issues/2727 and maybe other issues. Since the contents/fields are different from other files, I think it should be configured as a file collection item and excluded by default from a folder collection. (So I need to fix the issue 1.) Will dig into the topic further.

**Comment by kyoshino** [comment]
Sorry for the delay! I think I have figured out why the content is being ignored. When a Markdown file is used in a file collection, the `yaml-markdown` format is selected without content detection. But I think you’re primarily using `toml-markdown` for [your `_index.md`](https://github.com/onweru/website/blob/main/content/en/_index.md), so parsing the file fails. I’ll add format detection to solve the problem.

https://github.com/sveltia/sveltia-cms/blob/d5afb6f9c288a76eadf4301bc8d3819f5518661b/src/lib/services/contents/parser.js#L83-L87

**Reply by kyoshino** [reply, marked as answer]
The fixed landed in [v0.34.9](https://github.com/sveltia/sveltia-cms/releases/tag/v0.34.9) 🙇🏼

**Reply by onweru** [reply]
Thank you @kyoshino
